assets/queries/cloudFormation/aws/default_kms_key_usage/metadata.json

```
@@ -0,0 +1,11 @@
+{
+  "id": "e52395b4-250b-4c60-81d5-2e58c1d37abc",
+  "queryName": "Default KMS Key Usage",
+  "severity": "MEDIUM",
+  "category": "Encryption",
+  "descriptionText": "Ensure that storage is encrypted by KMS on instances that, based on their name, might host a database. ",
```

Not sure if this description fits the query name. When storage is encrypted, the default KMS Key is used. I think we should advise something like:

```suggestion
"descriptionText": "When `StorageEncrypted is set to true, `KmsKeyId` should not be undefined, to avoid the use of the default KMS Key",
```